hydra-configs-torch/hydra_configs/torch/optim/__init__.py

```diff
@@ -12,7 +10,12 @@
 from .sgd import SGDConf
 from .rprop import RpropConf
 from .rmsprop import RMSpropConf
+
+# from .optimizer import OptimizerConf
```

Comment:
Why the commented out code?

Reply:
stuff that we havent generated configs for yet. For example, this is the base optimizer. It's unlikely that anyone will need configs for this. I'll just remove these from the __init__.py for now.